lib/Playwright.pm

```diff
@@ -751,7 +752,7 @@ sub _start_server ( $port, $cdp_uri, $timeout, $debug, $cleanup ) {
     # Orphan the process in the event that cleanup => 0
     if (!$cleanup) {
         print "Detaching child process...\n";
-        chdir '/';
+        #chdir '/';
```

Comment:
Chdir into some dir is good for safety of daemonized processes.

I'll file an issue to send it into a File::Temp::tempdir instead.